packages/domain-picker/src/domain-picker/style.scss

```
@@ -109,6 +109,17 @@ $accent-blue: #117ac9;
 		}
 	}
 
+	&.type-link {
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+
+		@include break-medium {
+			flex-direction: initial;
```

This may be more explicit:

```suggestion
			flex-direction: row;
```

Is there a reason why you'd choose `initial` over `row`?

---

> Is there a reason why you'd choose `initial` over `row`?

Nope, it's just a matter of taste. Like we discussed here p1611667274030700-slack-GLMDTNTFD, we have to define conventions for this kind of situations. When it comes to `flexbox`, we have chosen to be more explicit to improve readability. I accepted your suggestion. Thanks 👍